src/main/java/edu/kis/powp/jobs2d/command/historyComponent/HistoryWindowCommandChangeObserver.java

```diff
@@ -0,0 +1,17 @@
+package edu.kis.powp.jobs2d.command.historyComponent;
+
+import edu.kis.powp.observer.Subscriber;
+
+public class HistoryWindowCommandChangeObserver implements Subscriber {
+
+    private HistoryManagerWindow backLogManagerWindow;
```

Comment:
Two things: 
1. This is a remain of plagiarism (#63).
2. You haven't managed to implement a suggestion:
"Try to redesign the history component to be independent of the GUI. (e.g. we might want to change GUI or use history from another component without accessing GUI)." 
Now, this is obligatory. There should be another way to add or take something from history than accessing GUI.

Reply:
We haven't changed this file since ours previous PR. (https://github.com/iis-powp-2020/powp_jobs2d_projects/pull/68)
I'm really sorry, but then it wasn't a plagiarism, but is it now?